.claude/skills/cmux-codex-autoapprove/scripts/cmux_codex_approve_launchd.py

```
@@ -49,146 +40,44 @@
-            if not structural_pass:
-                snippet = tail_text.replace("\n", " | ")[:120]
-                log(f"EMBED_SIM_MISS snippet={snippet!r}")
-        return semantic_pass and structural_pass
-    except Exception:
-        return bool(CANDIDATE_RE.search(tail_text) and BOTTOM_APPROVAL_RE.search(tail_text))
+def is_idle_prompt(line: str) -> bool:
+    """True if line is a bare shell prompt with no pending input."""
+    stripped = line.strip()
+    if not stripped:
+        return False
+    if stripped in {"❯", ">", ">>"}:
```

Ruff (0.15.9)

[warning] String contains ambiguous `❯` (HEAVY RIGHT-POINTING ANGLE QUOTATION MARK ORNAMENT). Did you mean `>` (GREATER-THAN SIGN)?

(RUF001)